products/hbase.md

```diff
@@ -19,8 +19,8 @@ releases:
 -   releaseCycle: "2.5"
     eol: false
     releaseDate: 2022-08-31
-    latestReleaseDate: 2023-02-01
```

Comment:
We should switch to GitHub for release notes (example : https://github.com/apache/hbase/releases/tag/rel%2F2.5.3).

Reply:
They just forgot to publish release notes for 2.5.4, no need to do anything.